packages/admin/admin/src/form/fields/SelectField.tsx

```diff
@@ -4,11 +4,13 @@ import { type ComponentProps, type ReactNode } from "react";
 import { Field, type FieldProps } from "../Field";
 import { FinalFormSelect } from "../FinalFormSelect";
 
-export type SelectFieldOption<Value extends string | number = string | number> = {
-    label: ReactNode;
-    value: Value;
-    disabled?: boolean;
-};
+export type SelectFieldOption<Value extends string | number = string | number> =
+    | {
+          label: ReactNode;
+          value: Value;
+          disabled?: boolean;
+      }
+    | string;
```

Comment:
this is an additional change I had to make:

Add plain string to supported options for RadioGroupField and SelectField to be compatible with StaticSelectValue from admin-generator
    
    This fix is required because the types of the fields where improved (fixed)

Reply:
But do we want this? Will there ever be a case where we want a string as both option label and value, where the label isn't translatable?

Before, the Admin Generator took care of converting the options: https://github.com/vivid-planet/comet/blob/main/packages/admin/admin-generator/src/commands/generate/generateForm/generateFormField.ts#L314-L325.